x/ecocredit/server/genesis.go

```diff
@@ -165,6 +168,17 @@ func setBalanceAndSupply(store sdk.KVStore, balances []*ecocredit.Balance) error
 		}
 	}
 
+	keys := make([]string, 0, len(batchDenomToBasketBalanceMap))
+	for k := range batchDenomToBasketBalanceMap {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+
+	for _, denom := range keys {
+		key := ecocredit.TradableSupplyKey(ecocredit.BatchDenomT(denom))
+		ecocredit.AddAndSetDecimal(store, key, batchDenomToBasketBalanceMap[denom])
```

Comment:
haven't looked at the genesis code in a while, so had a question here: why do we set it in the KVStore and not ORM?

Reply:
oh nvm i just saw the branch you're merging into lol. dont mind me 💩